web/src/user_group_edit.js

```diff
@@ -362,7 +427,77 @@ function update_toggler_for_group_setting() {
     toggler.goto(select_tab);
 }
 
+function get_subgroup_names_for_status_text(subgroups) {
+    const subgroup_names = subgroups.map((subgroup) => subgroup.name);
+    if (Intl.ListFormat === undefined) {
```

Comment:
This is somewhat repeated for what we have in `format_array_as_list`, but we need to have the names bold. Maybe can add a function like `format_array_list_with_highlighted_elements`.

Reply:
Yeah, probably such a function in `util.ts` would be a helpful abstraction.

Reply:
Moved this function to `util.ts` and updated variable names to be generic.